lib/grizzly/zwave/commands/configuration_properties_report.ex

```diff
@@ -178,9 +178,12 @@ defmodule Grizzly.ZWave.Commands.ConfigurationPropertiesReport do
 
     case format do
       :signed_integer ->
-        <<min_value::integer-signed-unit(8)-size(size),
+        <<
+          min_value::integer-signed-unit(8)-size(size),
+          # TODO: integer-signed-unit(8)-size(size) -> signed-size-unit(8) causes syntax error
```

Comment:
Looks like the suggestion is incorrect. `signed-size(size)-unit(8)` should work.

Reply:
Yup, I'm working on a PR for it.

Reply:
Fixed. We can wait for the update or I can do it in another PR. 😄